unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/Outfits/OutfitsSectionComponentView.cs

```diff
@@ -0,0 +1,212 @@
+using Cysharp.Threading.Tasks;
+using DCL;
+using DG.Tweening;
+using MainScripts.DCL.Controllers.HUD.CharacterPreview;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class OutfitsSectionComponentView : BaseComponentView, IOutfitsSectionComponentView
+{
+    [SerializeField] internal Button backButton;
+    [SerializeField] internal OutfitComponentView[] outfitComponentViews;
+    [SerializeField] private RawImage avatarPreviewImage;
+    [SerializeField] private GameObject discardOutfitModal;
+    [SerializeField] private Button confirmDiscardOutfit;
+    [SerializeField] private Button cancelDiscardOutfit;
+    [SerializeField] private Button closeDiscardOutfit;
+
+    private ICharacterPreviewController characterPreviewController;
+    private IUserProfileBridge userProfileBridge;
+    private DataStore dataStore;
```

Comment:
I dont think the view should know about these things. this is too tight to business logic to be here.